tests/test-python-project/python/parse_cpp_tree_test.py

```diff
@@ -32,12 +30,15 @@ def assert_cpp_files_similar(cpp_file1: str, cpp_file2: str):
         # both lists of tokens and pairwise similar
         assert token1.kind == token2.kind
         assert token1.spelling == token2.spelling
+        #
+        try:  #
+            token1 = next(tokens1)  #
+            token2 = next(tokens2)  #
+        except:  # comment
```

Comment:
E722 Do not use bare `except`
```suggestion
        except StopIteration:

```